src/main/features/claim/views/eligibility/single-claimant.njk

```diff
@@ -0,0 +1,40 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, errorSummary, radioGroup, saveAndContinueButton %}
+
+{% set title = t('Single Claimant | Money claim') %}
+
+{% block content %}
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <h1 class="heading-large">{{ t('Are you the only person or business making the claim?') }}</h1>
+      <p>{{ t('You can’t use this service if more than one person or business is making the claim') }}</p>
+
+      {% set name = 'singleClaimant' %}
+
+      <form novalidate method="post" class="analytics-submit-event-trigger"
+            data-event-action="Eligibility: Select Yes"
```

Comment:
This is not specific enough. A `Eligibility: Select Yes` is sent to GA so that we can see what people select. Seeing `Eligibility: Select Yes` only does not help us understand what Yes / No values mean. I would suggest something in line with `Eligibility: Single claimant`. Having this we will see the following in the GA:

```
Eligibility: Single claimant | Yes | 34
Eligibility: Single claimant | No  | 12
```

Reply:
Great explanation.
Ill add what you suggested